packages/aila/src/core/Aila.test.ts

```diff
@@ -285,4 +286,37 @@ describe("Aila", () => {
       expect(ailaInstance.lesson.plan.title).toBe(newTitle);
     }, 20000);
   });
+
+  describe("categorisation", () => {
```

Comment:
Just adding one example test here to show how we can pass in the mock categoriser during Aila initialisation